src/components/new-request/search.vue

```diff
@@ -103,11 +103,44 @@
       <!-- Change Name flow -->
       <template v-else-if="isChangeName">
         <BusinessLookupFetch />
-
+        <v-dialog  v-if="isSocietyDisabled" v-model="showSocietiesInfo"
+          :max-width="maxWidth"
+          persistent>
+          <v-card>
+              <v-icon
+                class="float-right"
+                md
+                color="primary"
+                @click="closeSocietyDialog()"
+              >
+              mdi-close
+              </v-icon>
+              <template>
+                <v-card-text class="mt-n12">
+                  <v-container fluid>
+                    <v-row
+                      no-gutters
+                      class="text-center"
+                    >
+                      <v-col cols="12">
+                        To change a name for a Society
+                      </v-col>
+                      <v-col cols="12">
+                        please use the Societies Online website
+                      </v-col>
+                      <v-col cols="12">
+                        <a href="https://www.bcregistry.ca/societies/">https://www.bcregistry.ca/societies/</a>
+                      </v-col>
+                    </v-row>
+                  </v-container>
+                </v-card-text>
+              </template>
+          </v-card>
+        </v-dialog>
         <template v-if="isNameChangeable">
           <!-- XPRO jurisdiction -->
           <Jurisdiction
-            v-if="isChangeNameXpro"
+            v-if="isChangeNameXpro && !isSocietyDisabled"
```

Comment:
Would it make more sense to move this check to the entire template block here (line 140)?

Reply:
I think this will work itself out when the final flow is fixed. Resolving.